app/src/main/java/com/github/epfl/meili/posts/forum/ForumActivity.kt

```diff
@@ -141,17 +141,12 @@ class ForumActivity : PoiActivity(R.layout.activity_forum, R.id.forum_activity),
         val title = editTitleView.text.toString()
         val text = editTextVIew.text.toString()
 
-        val post = Post(poi.uid, user.uid, title, timestamp, text)
+        val post = Post(poi.uid, user.uid, title, timestamp, text, bitmap != null)
 
         viewModel.addElement(post.postId(), post)
 
-        if (bitmap != null) {
-            executor.execute {
-                compressAndUploadToFirebase(
-                    "images/forum/${post.postId()}",
-                    bitmap!!
-                )
-            }
+        if (post.hasPhoto) {
+            executor.execute { compressAndUploadToFirebase("images/forum/${post.postId()}", bitmap!!) }
```

Comment:
We should store this path as a const val (since it will be easier to change and appears multiple times)